README.md

```diff
@@ -1,31 +1,118 @@
-Welcome to the KubeVirt Documentation. This welcome page is provided as
-the entrypoint to the different topics of this user-guide.
+# KubeVirt User-Guide
 
-Try it out
-==========
+[![Build Status](https://travis-ci.org/kubevirt/kubevirt.github.io.svg?branch=master)](https://travis-ci.org/kubevirt/kubevirt.github.io)
```

Comment:
This build status from travis is for the website repo, not related to the user-guide...

Reply:
Thanks for that catch.  I forgot to update that url.